src/main/java/kr/co/antoon/webtoon/facade/WebtoonStatusFacade.java

```diff
@@ -16,21 +17,21 @@
 public class WebtoonStatusFacade {
     private final WebtoonStatusService webtoonStatusService;
     private final WebtoonStatusCountService webtoonStatusCountService;
+    private final AntCoinService antCoinService;
 
     @Transactional
     public WebtoonStatusResponse saveOrUpdate(WebtoonStatusType status, Long userId, Long webtoonId) {
         statusCheck(userId, webtoonId);
+
         WebtoonStatusCount webtoonStatusCount = webtoonStatusCountService.findByWebtoonId(webtoonId)
                 .orElseGet(() -> webtoonStatusCountService.save(webtoonId));
         webtoonStatusCount.updateCount(status);
 
         status = WebtoonStatusType.of(status);
         webtoonStatusService.save(webtoonId, userId, status);
+        WebtoonStatusResponse response = new WebtoonStatusResponse(webtoonStatusCount, status);
 
-        return new WebtoonStatusResponse(
-                webtoonStatusCount,
-                status
-        );
+        return antCoinService.joinWebtoon(userId, webtoonId, response, status);
```

Comment:
```suggestion
        return antCoinService.joinWebtoon(joinWebtoonDto);
```
이런식으로 dto 객체를 만들어서 하는 건 어떠신가요??

Reply:
넵! 객채로 만들어서 전달하는 것이 더 깔끔하고 가독성도 좋을 것 같아용! 리뷰 감사합니다. 수정해서 올릴게여